config/default/manager/manager.yaml

```diff
@@ -86,6 +86,12 @@ spec:
               configMapKeyRef:
                 name: mac-range-config
                 key: RANGE_END
+          - name: CA_ROTATE_INTERVAL
```

Comment:
We should document the existence of these on the bottom of the README. I would keep it hard-coded environment variables for now, there is nothing pushing us to do it differently.